docs/api/plms.yaml

```diff
@@ -285,6 +285,926 @@ paths:
+              schema:
+                $ref: '#/components/schemas/Error'
+        408:
+          description: Request Timeout
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        500:
+          description: Internal Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /books/last-added:
+    get:
+      operationId: getLastAddedBooks
+      summary: Last Added Books
+      description: Get last added books.
+      parameters:
+        - in: query
+          name: count
+          schema:
+            type: number
+          description: Return list of last added books with given count.
+      tags:
+        - Books
+      responses:
+        200:
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Books'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        408:
+          description: Request Timeout
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        500:
+          description: Internal Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /books/random:
+    get:
+      operationId: getRandomBooks
+      summary: Random Books
+      description: Get random books.
+      parameters:
+        - in: query
+          name: count
+          schema:
+            type: number
+          description: Return list of random books with given count.
+      tags:
+        - Books
+      responses:
+        200:
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Books'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        408:
+          description: Request Timeout
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        500:
+          description: Internal Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /books/open-library:
```

Comment:
instead of having open-api in our endpoints, we should just add search

Reply:
will change it to /search